Analyzer/test/Tamas/submitCrabJobsData.py

```diff
@@ -45,12 +45,12 @@
 config.Data.unitsPerJob = 50
 config.Data.publication = True
 config.Data.outputDatasetTag = config.General.requestName
-config.Data.outLFNDirBase = '/store/user/tvami/HSCP'
+config.Data.outLFNDirBase = '/store/user/dapparu/HSCP/Analysis/CMSSW_10_6_27'
```

Comment:
hi @dapparu I think we should have a `/test/Dylan/` folder and have these files there. Under Tamas these configs should have my settings. Can you please do that?

Reply:
I'll do that sure. Sorry.